src/components/site/Search.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const Search = props => (
+  <form className="dg_search-container" id="search-bar">
+    <input
+      className="dg_search-text-field"
```

Comment:
className here i think should be `_dg_search-input`

Reply:
done